components/core/tools/docker-images/clp-env-base-manylinux_2_28/Dockerfile

```
@@ -1,6 +1,8 @@
 # syntax=docker/dockerfile:1
 
-FROM quay.io/pypa/manylinux_2_28_aarch64
+# Multi-arch base image (amd64, arm64, ...). Build with `--platform` to select the
+# target architecture; see build.sh (PLATFORM env var for local cross-builds).
+FROM quay.io/pypa/manylinux_2_28
```

Hadolint (2.14.0)

[warning] Always tag the version of an image explicitly

(DL3006)